cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting for bd")
+        db_conn = None
+
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                self.stdout.write("BD is not running...")
+                time.sleep(1)
+
+        self.stdout.write(self.style.SUCCESS("BD is running"))
```

Comment:
There is a typo in the message. It should be 'DB is running' instead of 'BD is running'.